### PR TITLE
IdentityServer4 added

### PR DESCRIPTION
**Pull request by iamkajal**
Old Identity web Project removed and added a web API Project to use as an identity service. IdentityServer4 Installed.

### CODE REVIEW

**Comment by iamkajal** [comment]
@ratanparai Bhai, It would be nice if you review this PR and give some feedback.

File: src/Services/Identity/Identity.API/Controllers/IdentityController.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Identity.API.Controllers
```
Comment (ratanparai) [inline src/Services/Identity/Identity.API/Controllers/IdentityController.cs]: Name space should follow `OpenCodeFoundation.eSchool.*` convention

File: src/Services/Identity/Identity.API/Identity.API.csproj [hunk 2]
```diff
@@ -1,9 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
+
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <RootNamespace>OpenCodeFoundation.eSchool.Identity.API</RootNamespace>
```
Comment (ratanparai) [inline src/Services/Identity/Identity.API/Identity.API.csproj]: Please keep the `RootNamespace` block. It helps `Visual Studio` to create new class/interface by appending the namespace.

File: src/Services/Identity/Identity.API/Startup.cs [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.HttpsPolicy;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
-namespace OpenCodeFoundation.eSchool.Identity.API
+namespace Identity.API
```
Comment (ratanparai) [inline src/Services/Identity/Identity.API/Startup.cs]: fix namespace. see previous comments

**Pull request merged**